### PR TITLE
18MS: Rusting of trains

### PR DESCRIPTION
**Pull request by perwestling**

### CODE REVIEW

File: lib/engine/game/g_18_ms.rb [hunk 1]
```diff
@@ -137,6 +145,21 @@ def tile_lays(entity)
 
         [{ lay: true, upgrade: true }, { lay: :not_if_upgraded, upgrade: false }]
       end
+
+      private
+
+      def rust(train, salvage_value)
```
Comment (tobymao) [inline lib/engine/game/g_18_ms.rb:151]: let's rename this rebate

Comment (tobymao) [reply lib/engine/game/g_18_ms.rb:151]: salvage_value -> rebate

File: lib/engine/game/g_18_ms.rb [hunk 2]
```diff
@@ -137,6 +145,21 @@ def tile_lays(entity)
...
+        rusted_trains = trains.select { |t| !t.rusted || t.name == train }
+        return if rusted_trains.empty?
+
+        rusted_trains.each do |t|
+          @bank.spend(salvage_value, t.owner)
+          t.rust!
+        end
+
+        @log << "-- Event: #{rusted_trains.uniq.join(', ')} trains rust --"
+        @log << "Corporations received a salvage value of #{format_currency(salvage_value)} per rusted train"
```
Comment (tobymao) [inline lib/engine/game/g_18_ms.rb:161]: Corporations receive a rebate of

Comment (perwestling) [reply lib/engine/game/g_18_ms.rb:161]: Salvage is the term used in the 18MS rules

**Pull request merged**